Include/Acidanthera/Library/OcMachoLib.h

```diff
@@ -329,7 +615,37 @@ MachoGetSymbolName64 (
   );
 
 /**
-  Retrieves Symbol's name.
+  Retrieves a Symbol's name.
+
+  @param[in,out] Context  Context of the Mach-O.
+  @param[in]     Symbol   Symbol to retrieve the name of.
+
+  @retval symbol name.
+
+**/
+CONST CHAR8 *
+MachoGetSymbolNameAny (
```

Comment:
The choice of `Any`-suffixed and "none"-suffixed versions (as opposed to `32`/`64`) for generic function names is a little worrying to me. Do we really need `Any` in most places we have it? Is there any logic behind `Any`?